test/data/mini_example/mini_example.sam

```diff
@@ -0,0 +1,69 @@
+read027	0	chr1	94	60	4M46S	*	0	0	ATTTGGATCTTGACTCTGGAAAACTTTTAACGCCGGGAATCGGTAGTCCT	*	AS:i:4	NM:i:0
+read023	0	chr1	101	60	25M15I10M	*	0	0	ATCGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTTGGATCTTGACT	*	AS:i:31	NM:i:15
+read024	0	chr1	102	60	24M15I11M	*	0	0	TCGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTTGGATCTTGACTC	*	AS:i:31	NM:i:15
+read025	0	chr1	103	60	23M15I12M	*	0	0	CGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTTGGATCTTGACTCT	*	AS:i:31	NM:i:15
+read021	2048	chr1	108	60	44S6M	*	0	0	GGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATTTTAAACGGCCCC	*	AS:i:6	NM:i:0
+read026	0	chr1	126	60	15S35M	*	0	0	CCCCGGGGCCAATTTGGATCTTGACTCTGGAAAACTTTTAACGCCGGGAA	*	AS:i:35	NM:i:0
+read027	0	chr1	126	60	4S46M	*	0	0	ATTTGGATCTTGACTCTGGAAAACTTTTAACGCCGGGAATCGGTAGTCCT	*	AS:i:46	NM:i:0	SA:Z:chr1,94,+,4M46S,60,4,0;
+read028	0	chr1	139	60	50M	*	0	0	GGAAAACTTTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTT	*	AS:i:50	NM:i:0
+read029	0	chr1	146	60	43M7S	*	0	0	TTTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATT	*	AS:i:43	NM:i:0	SA:Z:chr1,181,+,43S7M,60,7,0;
+read029	2056	chr1	146	60	35M8I7M	*	0	0	TTTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATT	*	AS:i:38	NM:i:8
+read030	0	chr1	147	60	42M8S	*	0	0	TTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATTT	*	AS:i:42	NM:i:0	SA:Z:chr1,181,+,42S8M,60,8,0;
+read030	2056	chr1	147	60	34M8I8M	*	0	0	TTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATTT	*	AS:i:38	NM:i:8
+read031	0	chr1	151	60	30M8I12M	*	0	0	ACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATTTTACG	*	AS:i:38	NM:i:8
+read031	2056	chr1	151	60	38M8I4M	*	0	0	ACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATTTTACG	*	AS:i:38	NM:i:8
+read032	0	chr1	180	60	1M8I41M	*	0	0	GATATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAAT	*	AS:i:38	NM:i:8
+read032	2056	chr1	180	60	9M8I33M	*	0	0	GATATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAAT	*	AS:i:38	NM:i:8
+read029	2048	chr1	181	60	43S7M	*	0	0	TTTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATT	*	AS:i:7	NM:i:0
+read030	2048	chr1	181	60	42S8M	*	0	0	TTTAACGCCGGGAATCGGTAGTCCTTTCGCGGGGATATATTTATATATTT	*	AS:i:8	NM:i:0
+read033	0	chr1	181	60	8S42M	*	0	0	ATATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAATA	*	AS:i:42	NM:i:0	SA:Z:chr1,181,+,8M42S,60,8,0;
+read033	2048	chr1	181	60	8M42S	*	0	0	ATATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAATA	*	AS:i:8	NM:i:0
+read033	2056	chr1	181	60	8M8I34M	*	0	0	ATATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAATA	*	AS:i:38	NM:i:8
+read034	0	chr1	181	60	7S43M	*	0	0	TATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAATAG	*	AS:i:43	NM:i:0	SA:Z:chr1,182,+,7M43S,60,7,0;
+read035	0	chr1	181	60	1S49M	*	0	0	TATATATTTTACGGGATATAACGATCGGATCGGATCGATAATAGGTCTCG	*	AS:i:49	NM:i:0
+read036	0	chr1	181	60	50M	*	0	0	ATATATTTTACGGGATATAACGATCGGATCGGATCGATAATAGGTCTCGG	*	AS:i:50	NM:i:0
+read034	2056	chr1	182	60	7M8I35M	*	0	0	TATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAATAG	*	AS:i:38	NM:i:8
+read034	2048	chr1	182	60	7M43S	*	0	0	TATATTTATATATTTTACGGGATATAACGATCGGATCGGATCGATAATAG	*	AS:i:7	NM:i:0
+read035	2056	chr1	188	60	1M8I41M	*	0	0	TATATATTTTACGGGATATAACGATCGGATCGGATCGATAATAGGTCTCG	*	AS:i:38	NM:i:8
```

Comment:
The alignment with this CIGAR string (1M8I41M) is unlikely because the long insertion gap costs much more than the one match is worth. Instead, the aligner would probably just clip the read (9S41M). I saw multiple instances of this, e.g. for read039 and read040.

Reply:
Yes you're probably right. As my example is really small, handling super short SVs, I thougt allowing this would be fine. 
I made similar things with the split reads: 2S48M and 2M48S. I thing the second is probably just not existent normally.
Should I change these somewhat unlikely CIGARs, or leave it anyway? My idea was to have some enclosed insertions etc.
But maybe you're right, in this case changing it to 9S41M would probably make more sense.

Reply:
Hm, you are right that the variants in the mini example are very short in order to keep the example simple and understandable. But maybe we can keep at least the relative sizes realistic. The reads in your example are 50bp long while real long reads have 5kb or more. How about this quick check: we multiply the CIGAR strings by 100 and check whether the resulting CIGAR string would be realistic.

When we take this CIGAR (7M8I35M) and multiply it by 100 we get 700M800I3500M which looks quite realistic. However, if we take the CIGAR (1M8I41M) that I pointed out and multiply it by 100 we get 100M800I4100M which is maybe less realistic.

I understand that you want to have some insertions that are completely covered but I think there are still enough of those if we remove all those alignments with only one match at the end and replace them with a soft-clipped version (e.g. 9S41M). Alignments with two or more matches at the end I would leave in the dataset for now.